microsoft-edge/progressive-web-apps-chromium/how-to/icon-theme-color.md

```diff
@@ -60,10 +60,219 @@ Each icon in the `icons` array should come with at least the `src` and `sizes` p
+    * 310x310 - large tile
+    * 388x388 - large tile 1.25x display scale
+    * 465x465 - large tile 1.5x display scale
+    * 620x620 - large tile 2x display scale
+    * 1240x1240 - large tile 4x display scale
+    * 50x50 - store tile
+    * 63x63 - store tile 1.25x display scale
+    * 75x75 - store tile 1.5x display scale
+    * 100x100 - store tile 2x display scale
+    * 200x200 - store tile 4x display scale
+    * 620x300 - splash screen
+    * 775x375 - splash screen 1.25x display scale
+    * 930x450 - splash screen 1.5x display scale
+    * 1240x600 - splash screen 2x display scale
+    * 2480x1200 - splash screen 4x display scale
+    * 16x16 - target size for taskbar, start menu, task manager
+    * 20x20 - target size for taskbar, start menu, task manager
+    * 24x24 - target size for taskbar, start menu, task manager
+    * 30x30 - target size for taskbar, start menu, task manager
+    * 32x32 - target size for taskbar, start menu, task manager
+    * 36x36 - target size for taskbar, start menu, task manager
+    * 40x40 - target size for taskbar, start menu, task manager
+    * 44x44 - target size for taskbar, start menu, task manager
+    * 48x48 - target size for taskbar, start menu, task manager
+    * 60x60 - target size for taskbar, start menu, task manager
+    * 64x64 - target size for taskbar, start menu, task manager
+    * 72x72 - target size for taskbar, start menu, task manager
+    * 80x80 - target size for taskbar, start menu, task manager
+    * 96x96 - target size for taskbar, start menu, task manager
+    * 256x256 - target size for taskbar, start menu, task manager
+In level 4, you supply images for tiles with display scales and target size images for display in various surfaces in Windows, including taskbar, start menu, task manager, ALT+Tab task switcher, and more.
+
+This provides the best experience for your users, but also requires the most developer effort.
+
+### Image Level Summary
+
+Consider, for example, how your app icon might show different content depending on the available surface area:
+
+* Small surface: App icon (44x44) in taskbar, showing a small, abbreviated logo: ![The small size icon](../media/windows-image-app-icon.png) 
+
+* Larger surface: Medium tile (150x150) in start menu, showing the full logo of the app:  ![The medium size icon.](../media/windows-image-small-tile.png) 
+
+* Wide surface: Wide tile in start menu, showing a more articulated app icon: ![The wide size icon](../media/windows-image-wide-tile.png) 
+
+Notice how the content of the icon changed. This is possible only through the use of higher levels of icon support. We recommend developers choose higher levels to give the best experience for your Windows users.  
+
+### Icon Descriptions
+
+Below you’ll find a description of each app icon and where they show up in Windows.
+
+![App Icon](../media/windows-image-app-icon.png) 
+The normal app icon that's shown in the start menu, task bar, or task manager.
```

Comment:
Could you please move the image below the text? It seems more logical to me to first read the icon description, and then see the image example below that.

Also, please add 1 empty line between the text and the image, in order to make sure the image is on its own line, and not next to the text.

Same comment for the other sections below.

Reply:
done!